src/ImageSharp/Memory/ArrayPoolMemoryManager.cs

```diff
@@ -30,16 +30,18 @@ public ArrayPoolMemoryManager(int minSizeBytes = 0)
         }
 
         /// <inheritdoc />
-        internal override Buffer<T> Allocate<T>(int size, bool clear = false)
+        internal override Buffer<T> Allocate<T>(int itemCount, bool clear = false)
```

Comment:
Let's use two distinct methods here with overloads. I'm not a big fan of using optional parameters in inherited methods; it can often lead to breaking changes since this could well become a public API.